packages/contracts/deploy-config/sepolia.json

```diff
@@ -14,6 +14,21 @@
   "validatorPoolRequiredBondAmount": "0x16345785d8a0000",
   "validatorPoolMaxUnbond": 10,
   "validatorPoolRoundDuration": 300,
+  "validatorPoolTerminateOutputIndex": "0x1F9A5",
```

Comment:
It seems to be different with that informed in our discord channel. Is this right? In discord, output index of 129446 is informed.

Reply:
Termination index is the last output index of ValidatorPool, and the index in the announcement is the first output index of ValidatorManager.

Reply:
Ah, I see.